app/templates/components/widget-math.hbs

```diff
@@ -0,0 +1 @@
+<span class="tex">{{{data.tex}}}</span>
```

Comment:
This is safe to render unescaped, I assume?

Reply:
Yes, I've tested it with `4 > 5` TeX syntax.